distro/BUILD

```diff
@@ -20,9 +20,8 @@ load("@rules_python//python:defs.bzl", "py_test")
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
 
-licenses(["notice"])
-
 package(
+    default_applicable_licenses = ["//:license"],
```

Comment:
This is more of a general comment. This change does not add a `default_applicable_licenses = ["//:license"]` declaration to the following three BUILD files. Should it?
```
tests/mappings/external_repo/pkg/BUILD
deb_packages/tools/update_deb_packages/src/BUILD
deb_packages/BUILD
```

Reply:
deb_packages is not part of the distribution. It is effectively not part of the rules_pkg at all, now that it has forked and other are continuing development.

test/mappings/external_repo/pkg/BUILD can not point at the license because it is a distinct repo. It also is not used by users, so it never goes into an end users code.  I don't feel a need to solve this problem now. What I want to do is build a integration test that builds a new WORKSPACE dynamically, the way distro tests do.

Reply:
I see. Thank you for the clarifications Tony